### PR TITLE
app-dir

### PR DESCRIPTION
**Pull request by jonthegeek**
### Common Mistakes to Catch Fast:
- [x] Did you increment the version number?
- [x] Did you re-build and document the package at the very end to make sure all changes were picked up?
- [x] Does `goodpractice::gp()` come up clean?

### Tests:
- [x] Do you have tests for all of the corner cases you can think of?

### Documentation:
- [x] Are all functions documented, including arguments and return values?
- [x] Is there at least one example in the documentation of each exported function?
- [x] Did you update the README (or conciously decided not to do so)?

### Code Quality:
- [x] Are all parameters validated?
- [x] Is the parameter that's most likely to be piped into this function the first parameter?
- [x] Are all internal functions prepended with '.'?
- [x] Is there anything hacky that you meant to address before PRing?

### Housekeeping:
- [ ] Did you review the author checklist (especially things in bold)?

### Documentation:
- [ ] Can you understand what this feature is supposed to do from its help?
- [ ] Does the example seem like a typical use case?

### Code Quality:
- [ ] Can you easily determine what each section of code does?

### Tests:
- [ ] Can you build the package with your personal Rstudio setup?
- [ ] Do the tests pass for you?
- [ ] Can you think of any missing tests?

### CODE REVIEW

File: R/path.R [hunk 1]
```diff
@@ -0,0 +1,21 @@
+#' Path to an App Cache Directory
+#'
+#' App cache directories can depend on the user's operating system and an
+#' overall \code{R_USER_CACHE_DIR} environment variable. This function
```
Comment (jonathanbratt) [inline R/path.R:4]: incomplete thought?

**Review by jonathanbratt — CHANGES_REQUESTED** [review]
looks like some documentation is incomplete

**Review by jonathanbratt — APPROVED** [review]
looks good!
